### PR TITLE
Updating docker image name

### PR DESCRIPTION
**Pull request by jeremyrickard**
Migrates name from azure-service-broker to azure-open-service-broker

Fixes: #100

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,11 +3,7 @@ GIT_VERSION = $(shell git describe --always --abbrev=7 --dirty)
...
 BINARY_NAME := osba
 
-# This is left as 'azure-service-broker' because we don't yet have a docker repo
-# for 'open-service-broker-azure'
-#
-# See https://github.com/Azure/open-service-broker-azure/issues/100
-BASE_IMAGE_NAME        = azure-service-broker
+BASE_IMAGE_NAME        = azure-open-service-broker
 
 RC_IMAGE_NAME          = $(REGISTRY)$(BASE_IMAGE_NAME):$(GIT_VERSION)
```
Comment (carolynvs) [inline Makefile]: Where did we finally land on setting `REGISTRY`? In circle?

Comment (carolynvs) [reply Makefile]: Nevermind just saw your other PR!

**Comment by krancour** [comment]
idk why naming things has been such a challenge for us. I hate having to ask this...

The name we'd _wanted_ for the Docker Hub repo would be one that matches the name of this GitHub repo: `open-service-broker-azure`.

But, after a lot of back and forth, the name they gave us was `azure-open-service-broker`...

Since this isn't the name we asked for, I have to ask whether there's any _actual_ benefit of switching from the old Docker Hub repo name (`azure-service- broker`) to this...

They're both wrong. So should we keep the nicer of the two until we get the name we wanted?

**Comment by seanmck** [comment]
Agree with @krancour. Hold off on this for now - I didn't have a chance to engage much on this topic in the run-up to KubeCon. Will connect with Steve and see if we can get alignment.

**Pull request closed without merging**